src/network/networkd-link.c

```diff
@@ -2092,6 +2092,7 @@ static int link_up_can(Link *link) {
 }
 
 static int link_set_can(Link *link) {
+        struct can_ctrlmode cm = { 0 };
```

Comment:
Please drop `0` here. `= {};` is enough.

Reply:
yeah, @yuwata is right, the `{0}` thing is a c99ism we don't bother with, as gnu allows omition of all initializers, and hence accepts `{}` too